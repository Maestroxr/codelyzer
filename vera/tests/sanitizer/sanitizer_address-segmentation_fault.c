/* SEGMENTATION FAULT */
#include<stdio.h> 
#include<stdlib.h> 
#include<string.h> 

int main(void) 
{ 
	char* p; 

	p[ 100 ] = 'a';
	printf("\n %s \n", p); 

	return 0; 
}