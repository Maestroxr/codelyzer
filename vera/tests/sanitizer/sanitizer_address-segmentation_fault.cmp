C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault.c:8: Uninitialized identifier 'p' of type char*.
