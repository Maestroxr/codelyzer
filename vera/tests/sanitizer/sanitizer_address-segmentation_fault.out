C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault.c:8: Uninitialized identifier 'p' of type char*.
C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault.c:10: Spacing for identifier/literal in operator [], supposed to be '[ 100 ]'.
C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault.c:10: Memory:[C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault]: Segmentation fault, suspect variables: ['p'] at:
		#2 sanitizer_address-segmentation_fault.c:10 in function 'main'
